generator/src/test/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGeneratorTest.java

```diff
@@ -110,4 +101,30 @@ public void testGeneratesValidExpressions() throws Exception {
     }
   }
 
+  private List<Stmt> makeVariableListAndAssignMutatedExpression(BasicType basicType,
+                                                                ShadingLanguageVersion shadingLanguageVersion,
+                                                                int variableNumber) {
+    final List<Stmt> newStmts = new ArrayList<>();
+    final GenerationParams generationParams = GenerationParams.large(ShaderKind.FRAGMENT, true);
+    // variableNumber should be large enough to get high coverage, e.g. 1000
```

Comment:
Please move this comment to the start of the method (first line after the "{"), and change the name of the referenced parameter accordingly.

Reply:
Done!